_posts/2021-11-18-keras-updates/keras-updates.Rmd

```diff
@@ -0,0 +1,159 @@
+---
+title: "Keras for R is back!"
+description: > 
+  For a while, it may have seemed that Keras for R were in some undecidable state, like Schrödinger's cat before inspection. It is high time to  correct that impression. Keras for R is back, with two recent releases adding powerful capabilities that considerably lighten previously tedious tasks. 
+  This post provides a high-level overview. Future posts will go into more detail on some of the most helpful new features, as well as dive into the powerful low-level enhancements that make the former possible.
```

Comment:
I don't know if you want so much on the future posts in the _description_ for this post. It certainly belongs in the post itself (in the intro or whatever, and then add links to future posts when they come), but might be pared down here.

Reply:
this is because i'm gonna lead with the book (this post has some kind of exceptional structure, i know - but need to experiment sometimes, too :-)))